packages/core/admin/server/src/controllers/admin.ts

```diff
@@ -154,12 +154,18 @@ export default {
   async plugins(ctx: Context) {
     const enabledPlugins = strapi.config.get('enabledPlugins') as any;
 
-    const plugins = Object.entries(enabledPlugins).map(([key, plugin]: any) => ({
-      name: plugin.info.name || key,
-      displayName: plugin.info.displayName || plugin.info.name || key,
-      description: plugin.info.description || '',
-      packageName: plugin.info.packageName,
-    }));
+    // List of core plugins that are always enabled,
+    // and so it's not necessary to display them in the plugins list
+    const CORE_PLUGINS = ['content-manager', 'content-type-builder', 'email', 'upload', 'i18n'];
```

Comment:
I think the official term now is core packages

Reply:
shouldn't we rename everything in here "packages"? even the configuration references plugins, and it feels "off" to reference packages in this const variable but reference plugins in the rest of the controller.